CHANGES.rst

```diff
@@ -42,6 +42,9 @@ New Features
 
 - ``astropy.io.fits``
 
+  - Support reading and writing from bzip2 compressed files. i.e. ``.fits.bz2``
+    files.
```

Comment:
One little nitpick is to include the issue number here.  After the period just add a space followed by `[#3789]` (per the example of other changelog entries)